ziti/contexts.go

```diff
@@ -107,41 +104,21 @@ func NewContextWithOpts(cfg *config.Config, options *Options) (Context, error) {
 		closeNotify:       make(chan struct{}),
 	}
 
-	ztUrl, err := url.Parse(cfg.ZtAPI)
-
-	if err != nil {
-		return nil, err
+	if cfg.ID.Cert != "" && cfg.ID.Key != "" {
```

Comment:
what if we started to add godoc to public functions and start here? this little bit of logic is really important to understand as a consumer

Reply:
Sure.